google/cloud/bigquery/client.py

```diff
@@ -1654,7 +1657,6 @@ def create_job(self, job_config, retry=DEFAULT_RETRY):
             )
         elif "query" in job_config:
             copy_config = copy.deepcopy(job_config)
-            _del_sub_prop(copy_config, ["query", "destinationTable"])
```

Comment:
I didn't realize we were already removing this property in past releases. We should restore this, since I don't want to break existing clients.